### ISSUE TITLE
rwe/launcher/src/master/master-server.ts - Non recommended port in use

### ISSUE THREAD

**Issue by Arpaesis** [MHeasell/rwe#60]
The default port is set to 5000, though this port is not recommended, as it is registered under the IANA by a multitude of entities. Using a different port is recommended, one that isn't registered preferably. 👍

**Comment by MHeasell** [comment]
I don't think this really matters at the moment because that port is only actually used in development -- in production the master server is reachable over the normal HTTPS port.

It might matter more when local game servers are supported. Feel free to suggest a different port number to use.
